verification/cbmc/proofs/aws_hash_table_clean_up/aws_hash_table_clean_up_harness.c

```diff
@@ -27,6 +27,7 @@ void aws_hash_table_clean_up_harness() {
     // This would normally raise a warning since the memory was deallocated, so use the pragma.
 #pragma CPROVER check push
 #pragma CPROVER check disable "pointer"
+#pragma CPROVER check disable "pointer-overflow"
```

Comment:
Can we create a GitHub issue to review all CBMC pragmas in AWS C Common?

Reply:
+1!

I would strongly suggest removing all of them from the sources and the proofs.

Reply:
https://github.com/awslabs/aws-c-common/issues/803

Reply:
Thanks!